content/docs/dev-and-releng/code-signing/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Codesigning for Windows binaries
+linkTitle: Codesigning
+description: >
+  We distribute signed CLI binaries (.exe) for Windows. Here is how to configure CI and the CLI repository, and how to update the certificate once it expires.
+---
+
+## Ensure creation of signed binaries
+
+Distributing a signed binary (aka codesigning) requires the following things. If all is in place, signed Windows binaries are created with every release. If something is missing, unsigned binaries are created instead.
```

Comment:
> `If something is missing, unsigned binaries are created instead.`

Sounds scary as one will likely not be able to run those unsigned binaries on Windows. Shouldn't the build/release process *fail* instead (if no `unsigned` or `test` flag is provided)?

Reply:
They do run, but the user gets an additional warning, and depending on the privileges, might not be able to execute it. That's not "scary" to me, just inconvenient.

Reply:
Depends who's the end user. If it's customers, I'd rather have the build fail in such cases, since I doubt there is an expiry warning being monitored for this certificate.

How to we know which projects use a certain signing certificate, when the time for rotation comes?

Reply:
These are all good questions which rather touch on process improvements than on docs improvements. I'll make a note in the Rainbow board.